src/lib/yaraparse.py

```diff
@@ -148,11 +154,30 @@ def ursify_plain_string(string: PlainString) -> UrsaExpression:
         return ursa_ascii
 
 
+def ursify_xor_string(string: PlainString) -> UrsaExpression:
+    text_ascii = string.pure_text
+    xored_strings: List[UrsaExpression] = []
+
+    for xor_key in range(256):
```

Comment:
Maybe add a comment like

```
# TODO(#100) implement modifier ranges

Reply:
(#100 is the issue number. Not sure about the reference, we can also use a full link to be more explicit)